src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs

```diff
@@ -79,7 +79,8 @@ impl Container {
         // process oci spec
         let mut inner = self.inner.write().await;
         let config = &self.config;
-        amend_spec(&mut spec).context("load spec")?;
+        let mut sandbox_pidns = false;
+        amend_spec(&mut spec, &mut sandbox_pidns).context("load spec")?;
```

Comment:
Suggest to return the `sandbox_pidns` by amend_spec not pass a `mut sandbox_pidns`.